teams/bigdata/membership.json

```diff
@@ -1,21 +1,27 @@
 {
-    "_comment": [
-        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    "repositories": [
+        "tidb-incubator/TiBigData",
+        "tispark",
+        "tispark-test",
+        "tispark-test-data"
+    ],
+    "reviewers": [],
+    "committers": [
+        "Coderlxl",
+        "LittleFall",
+        "ilovesoup",
+        "windtalker"
     ],
-    
     "maintainers": [
         "birdstorm",
         "humengyu2012",
         "marsishandsome",
         "sunxiaoguang"
     ],
-
-    "committers": [
-        "Coderlxl",
-        "LittleFall",
-        "ilovesoup",
-        "windtalker"
+    "description": [
```

Comment:
could be a string?

Reply:
Just follow the `_comment` style. I don't understand why `_comment` is a list.

Reply:
You're right 😄 . We should correct it later or if you don't mind it, include in this PR.